tutorials/Forecasting.ipynb

```diff
@@ -167,52 +163,39 @@
       },
       "outputs": [],
       "source": [
-        "item_id = \"<item_id>\" #change for the you item_id\n",
-        "starting_date = \"<starting_date>\" # example \"2021-08-07T07:14:19Z\"\n",
-        "reading_data_request = {\n",
-        "  \"items\": {\n",
-        "    \"include\": True,\n",
-        "    \"filter\": {\n",
-        "      \"id\": {\n",
-        "        \"$in\": [\n",
-        "          item_id\n",
-        "        ]\n",
-        "      }\n",
-        "    }\n",
-        "  },\n",
-        "  \"data\": {\n",
-        "    \"include\": True,\n",
-        "    \"notBefore\": starting_date #starting from \n",
-        "  }\n",
-        "}\n",
+        "item_id = \"c5jdi4jsbu8cohpla9vg\" #change for the you item_id\n",
```

Comment:
Maybe have it as <item_id> instead if the actuall id, with an example. The same with the starting_date.